src/update.c

```diff
@@ -675,6 +682,15 @@ enum swupd_code update_main(int argc, char **argv)
 		ret = check_update();
```

Comment:
Not sure how useful it is, but for me it's a bit weird to have `swupd udpate --status --3rd-party` not running `swupd 3rd-party update --status` like any other parameter

Reply:
cool, I didn't add it at first because I wanted to minimize the swupd commands that call a 3rd-party command, but I'll add it so it is consistent